Examples/ArcGISToolkitExamples/VCListViewController.swift

```diff
@@ -25,30 +25,17 @@ open class VCListViewController: TableViewController {
         }
     }
     
-    override open func viewDidLoad() {
-        super.viewDidLoad()
-    }
-    
-    required public init?(coder aDecoder: NSCoder) {
-        super.init(coder: aDecoder)
-    }
-    
-    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
-        super.init(nibName: nil, bundle: nil)
-    }
-    
     override open func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return viewControllerInfos.count
     }
     
     override open func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: cellReuseIdentifier)!
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
```

Comment:
Why not make use of `cellReuseIdentifier`?

Reply:
Since the class no longer inherits from `TableViewController`, `TableViewController. cellReuseIdentifier` is no longer available.

We could define our our property for the reuse identifier, but since it is only needed in one place (since the storyboard handles registering the cell), it doesn't make much difference either way.

Reply:
Oh :/ I hadn't noticed it stopped inheriting. I'd better pay more attention.